alpine/resin

```diff
@@ -0,0 +1,66 @@
+#!/sbin/openrc-run
```

Comment:
openrc has manpages you can check https://github.com/OpenRC/openrc/tree/master/man